### PR TITLE
Setting Next.js Backend with MongoDB Integration and Project Seed Data 

### PR DESCRIPTION
**Pull request by Ifechukwudaniel**
## Description
In the changes below, I have made significant progress in implementing the backend for the project. The updates include:

1. Integration of MongoDB using Mongoose ORM and Typegoose.
2. Implementation of POST and GET routes for the Projects Model.
3. Utilization of a list of project optimism planes to funds as seed data for our project data.

## Additional Information
I am submitting this pull request to seek feedback on whether I am heading in the right direction with the project. I would like to know what areas I should focus on improving. For this implementation, I am using the same architecture that was employed in [buidlguidl-v2](https://github.com/scaffold-eth/buidlguidl-v3).

Your ENS/address: ifechukwudaniel.eth

- [x]  I have read the [contributing docs](https://github.com/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)

- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

### CODE REVIEW

**Review by escottalexander — APPROVED** [review]
🔥 This looks good! just resolve the merge conflicts and we should be good!

File: packages/nextjs/local_database/importSeed.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// import fs from "fs";
+
+export async function importSeed() {
+  //   console.log("jdjjdjd");
```
Comment (damianmarti) [inline packages/nextjs/local_database/importSeed.ts:4]: Why is this function commented on?

Comment (Ifechukwudaniel) [reply packages/nextjs/local_database/importSeed.ts:4]: I aimed to develop a function that reads the project.json file and uses its data to seed the local database with projects. This way, the front-end team will have easy access to relevant data for their work.

**Comment by emudoteth** [comment]
> 🔥 This looks good! just resolve the merge conflicts and we should be good!

9s
Run yarn next:lint --max-warnings=0
Attention: Next.js now collects completely anonymous telemetry regarding usage.

./lib/dbConnect.ts
This information is used to shape Next.js' roadmap and prioritize features.
[4](https://github.com/scaffold-eth/OP-RetroPGF3-Discovery-Voting/actions/runs/5723912438/job/15569738288#step:6:5)0:30  Warning: Forbidden non-null assertion.  @typescript-eslint/no-non-null-assertion
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:

https://nextjs.org/telemetry
info  - Need to disable some ESLint rules? Learn more here: https://nextjs.org/docs/basic-features/eslint#disabling-rules

**Pull request closed without merging**